src/js/index.js

```diff
@@ -1 +1,34 @@
 import "../styles/index.css";
+import { startingPage } from "./starting-page";
+import { firstChapter, secondChapter, thirdChapter } from "./demo-story";
+import { createChapterStructure } from "./chapterDOM";
+import { render } from "./render";
+
+//Now it's just one story object, later we should create factory for stories
+const storyObj = {
+  chapters: [],
+  addChapter(...chapterData) {
```

Comment:
`addChapters` instead of `addChapter` would describe this function better.

Reply:
Totally agree.